docs/source/techniques/ISISPowder-Polaris-v1.rst

```diff
@@ -402,6 +412,30 @@ Example Input:
 
   polaris_example.focus(do_van_normalisation=True, ...)
 
+.. _van_normalisation_method_isis-powder-diffraction-ref:
+
+van_normalisation_method
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates whether a relative or absoluate normalisation should
+be performed. The possible values are "Relative" and "Absolute".
+
+This parameter is optional. The default value when mode="Rietveld" is "Relative". The default value when mode="PDF" is "Absolute".
+
+If "Absolute" is selected then the measured intensity is multipled by the following additional factor:
+
+.. math::
+
+    \frac{\rho_v V_v \sigma_v}{4 \pi \rho_s V_s}
```

Comment:
Is it worth adding the reference for this?? Feel free to say no!

Reply:
Yes I'll put one in from the Howe paper that we've been occasionally referring to in our discussions with Helen